documentation/database/sqlserver.md

```diff
@@ -139,28 +139,30 @@ Example: `jdbc:sqlserver://<i>host</i>:<i>port</i>;databaseName=<i>database</i>;
 
 ### Azure Active Directory
 
-#### Installing ADAL4J
+#### Installing MSAL4J
 
-To use Flyway with Azure Active Directory connections, you must install [ADAL4J](https://github.com/AzureAD/azure-activedirectory-library-for-java) and it's dependencies.
+To use Flyway with Azure Active Directory connections, you must install [MSAL4J](https://github.com/AzureAD/microsoft-authentication-library-for-java) and it's dependencies.
 
-If you're using Flyway in an environment that is integrated with Maven or Gradle (Like the Maven or Gradle plugin), you only need to add [ADAL4J's Maven package](https://mvnrepository.com/artifact/com.microsoft.azure/adal4j) as a dependency in your `pom.xml`. Maven should then deal with getting ADAL4J's dependencies itself.
+If you're using Flyway in an environment that is integrated with Maven or Gradle (Like the Maven or Gradle plugin), you only need to add [MSAL4J's Maven package](https://mvnrepository.com/artifact/com.microsoft.azure/msal4j) as a dependency in your `pom.xml`. Maven should then deal with getting MSAL4J's dependencies itself.
 
-For command line users, you'll have to download them all manually. To do this, you can use the JAR Download tool to find ADAL4J and its dependencies:
+For command line users, you'll have to download them all manually. To do this, you can use the JAR Download tool to find MSAL4J and its dependencies:
 
 - Go to the [JAR Download Tool](https://jar-download.com/online-maven-download-tool.php)
 - Paste the following into the `Maven XML` box:
 
 ```xml
 <dependency>
-  <groupId>com.microsoft.azure</groupId>
-  <artifactId>adal4j</artifactId>
-  <version>1.6.6</version>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>msal4j</artifactId>
+    <version>1.9.1</version>
 </dependency>
 ```
 
 - Click 'Submit'
 - Download the archive and add it to the `lib` folder in your command-line installation.
 
+ADAL4J is no longer [receiving improvements](https://github.com/AzureAD/azure-activedirectory-library-for-java) from Microsoft. 
```

Comment:
I don't think we need this line, since AFAL4J is not mentioned anywhere else, it's a bit out of place

Reply:
I've removed it again \o/